docs/src/tutorials/Add-New-Network.md

```diff
@@ -0,0 +1,164 @@
+# Add New Network
+
+Parity Signer supports adding any substrate-based networks or updating and existing network via QR code.
+
+After you've installed [required software](#Prerequisites), you need to add *Network's Specs* to Signer and add *Network Metadata* for this network, so that Signer is able to decode, and you could read and verify transactions you are signing on this network.
+
+If you need to update metadata for already existing network you only need to update *Network Metadata*.
+Off-the-shelf Signer comes with networks that you can update by scanning a multipart QR codes that containt recent metadata for these networks at [Metadata Update Portal](https://metadata.parity.io/).
+
+*Network Specs*
+
+1. [Get](#get-network-spec)
+2. [Sign](#sign-network-spec)
+3. [Feed into Signer](#feed-network-spec-into-signer)
+
+*Network Metadata*
+
+4. [Get](#get-network-metadata)
+5. [Sign](#sign-network-metadata)
+6. [Feed into Signer](#feed-network-metadata-into-signer)
+
+### Prerequisites
+
+- [ ] Websocket URL of the network (we maintain an [address book](https://github.com/paritytech/parity-signer/blob/master/rust/generate_message/address_book))
+- [ ] [rust](https://www.rust-lang.org/tools/install)
+- [ ] [subkey](https://docs.substrate.io/v3/tools/subkey/#installation)
+- [ ] a clone of [parity-signer repository](https://github.com/paritytech/parity-signer). In the repository go to `generate_message` crate:
+
+```
+cd rust/generate_message
+cargo build
+cargo run restore_defaults
+```
+
+With `subkey` you need to generate a private and public keys (`<secret-phrase>` and `Public key (hex)`). You will be able to update a network only with metadata that is signed with the same keys as network specs that defined the network for your Signer instance.
```

Comment:
Another alternative is to use development seed phrase with a `///password` - this way certificate will be as strong as password and have arbitrary format easy to store in any pw manager. This should be sufficient for personal use (that is, if the updates are not distributed to other users). I would ask for @kirushik to confirm this statement first though.

Reply:
Also note here that a dedicated Signer could be used to sign metadata; maybe even label it as "TODO: instructions" so that we release this update asap first. But users should be aware that this is possible IMO.